cmd/gtk/widget_wallet.go

```diff
@@ -136,9 +138,10 @@ func buildWidgetWallet(model *walletModel) (*widgetWallet, error) {
 		})
 
 	signals := map[string]interface{}{
-		"on_new_address":     w.onNewAddress,
-		"on_change_password": w.onChangePassword,
-		"on_show_seed":       w.onShowSeed,
+		"on_new_address":           w.onNewAddress,
+		"on_new_validator_address": w.onNewValidatorAddress,
```

Comment:
Introduce new singnal(on_new_validator_address) for creating new validator address